Feat(admin): solicita confirmação para deletar meditação

ThaisMap opened the pull request:
- Adiciona diálogo pedindo confirmação antes de deletar uma meditação
- Adiciona props para passar título e labels para os botões do diálogo

ThaisMap committed:
feat: adiciona props no dialogo

- permite passar props com título e labels para os botões

ThaisMap committed:
feat(admin): confirmação para deletar meditação

- Adiciona diálogo pedindo confirmação antes de deletar uma meditação

kellynvd commented:
@ThaisMap ,só  pra deixar registrado aqui o que a gente conversou: uma sugestão seria criar um componente (ex: ConfirmIconButton) com o _icon button_ e o _dialogo_. Esse componente faria o controle do abrir e fechar a modal. Dessa forma podemos reutiliza-lo e também não precisariamos do state `meditacaoSelecionada`

ThaisMap committed:
feat: adiciona componente IconButtonConfirmacao

- O componente contém um IconButton e um Diálogo para confirmar a ação

ThaisMap committed:
feat: usa iconButtonConfirmacao

- Troca o uso direto do Dialogo pelo uso do IconButtonConfirmacao para confirmar exclusão
